evap/staff/importers.py

```diff
@@ -510,6 +526,8 @@ def process_contributors(self, course, test_run, user_list):
         users_to_add = [user for user in user_list if user not in already_related]
 
         if not test_run:
+            self.make_users_active(user_list)
```

Comment:
do you need to do this at all? it says this here above
```
for user_data in self.users.values():
__, created = user_data.store_in_database()
```
and as far as i understand, this should set every user's `is_active` to True, or might any user from the excel file be missing?